app/util/YouTube.scala

```diff
@@ -20,6 +20,10 @@ trait YouTube extends Logging with YouTubeAccess with YouTubeVideos with YouTube
     cache.getOrElse("channels", duration) { super.channels }
   }
 
+  override def channelsWithData(setAllVideosPublic: Boolean): List[YouTubeChannelWithData] = {
```

Comment:
Do we need this since it's just forwarding on to the superclass?

Reply:
No we don't. It is gone.